src/test/java/org/folio/circulation/support/JsonPropertyFetcherTest.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.circulation.support;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimePropertyByPath;
+import static org.folio.circulation.support.JsonPropertyWriter.writeByPath;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class JsonPropertyFetcherTest {
+
+  @Test
+  public void shouldReturnNullIfSomeObjectsNotExistInPath() {
+    final String[] paths = {"1", "2", "3", "4", "5"};
+    final JsonObject object = new JsonObject()
+      .put("1", new JsonObject().put("2", new JsonObject()));
+
+    final DateTime actualValue = getDateTimePropertyByPath(object, paths);
+
+    assertThat(actualValue, nullValue());
+  }
+
+  @Test
+  public void shouldReturnDateTimePropertyByPath() {
+    final String[] paths = {"1", "2", "3", "4", "5"};
+    final DateTime expectedDate = DateTime.parse("2020-12-12T20:00:00.123Z");
+
+    final JsonObject object = new JsonObject();
+    writeByPath(object, expectedDate, paths);
+
+    final DateTime actualValue = getDateTimePropertyByPath(object, paths);
+    assertThat(actualValue, is(expectedDate));
+  }
```

Comment:
Similarly, I might be tempted to move this to the beginning of the tests as it is the simpler case